desc/equilibrium/equilibrium.py

```diff
@@ -638,9 +638,6 @@ def get_surface_at(self, rho=None, theta=None, zeta=None):
             Zb = AZ @ self.Z_lmn
             surface.R_lmn = Rb
             surface.Z_lmn = Zb
-            surface.grid = LinearGrid(
-                rho=rho, M=2 * surface.M, N=2 * surface.N, endpoint=True, NFP=self.NFP
-            )
```

Comment:
The surface classes no longer have grid attributes. This is a relic that we forgot to remove before.